package.json

```diff
@@ -3,13 +3,14 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "install": "lerna bootstrap",
+    "install": "lerna bootstrap --concurrency=2",
```

Comment:
Added to [prevent build memory failures](https://circleci.com/gh/getndazn/dazn-lambda-powertools/1023). 

Please see this related [Lerna issue](https://github.com/lerna/lerna/issues/1798).

Reply:
ha, that's a good find, think we removed the concurrency because it caused some other issue before, maybe we had the concurrency too high at the time